pkg/blockatlas/marketdata.go

```diff
@@ -0,0 +1,58 @@
+package blockatlas
+
+import (
+	"math/big"
+	"time"
+)
+
+const (
+	TypeCoin  CoinType = "coin"
+	TypeToken CoinType = "token"
+)
+
+type CoinType string
+
+type TickerResponse struct {
+	Currency string  `json:"currency"`
+	Result   Tickers `json:"result"`
+}
+
+type Ticker struct {
+	Coin       uint        `json:"coin"`
```

Comment:
can we add `CoinType` as `uint` type so we can reuse across different models

Reply:
```
type CoinType uint
const (
	TypeCoin  CoinType = 1
	TypeToken CoinType = 2
)
```

like that?

Reply:
I was thinking to map somehow with this value: https://github.com/trustwallet/blockatlas/blob/master/coin/coins.go#L13

but not sure if that's possible

Reply:
lgtm either way

Reply:
Example from iOS:

<img width="591" alt="Screen Shot 2019-11-12 at 5 10 50 PM" src="https://user-images.githubusercontent.com/1641795/68724079-60b1df80-056f-11ea-8c26-5916390be49d.png">

Reply:
we can do that, but we need to refactor the coin generator, it's easy, but a big change for this PR

Reply:
I miss enums

Reply:
yes :( me too